llvm/lib/MC/MCDwarf.cpp

```diff
@@ -185,29 +192,23 @@ void MCDwarfLineTable::emitOne(
   for (const MCDwarfLineEntry &LineEntry : LineEntries) {
     MCSymbol *Label = LineEntry.getLabel();
     const MCAsmInfo *asmInfo = MCOS->getContext().getAsmInfo();
-    if (LineEntry.IsEndEntry) {
+    if (LineEntry.IsEndEntry || LineEntry.IsEndOfFunction) {
       MCOS->emitDwarfAdvanceLineAddr(INT64_MAX, LastLabel, Label,
                                      asmInfo->getCodePointerSize());
       init();
+
```

Comment:
This is a nitpick, but it may save us a merge conflict if we don't have this empty line on the commit, since upstream this line doesn't exist.

Reply:
Yep, that is something I can clean up